ChaosMod/Effects/EffectsInfo.h

```diff
@@ -408,104 +409,105 @@ const std::map<EffectType, EffectInfo> g_effectsMap =
+	{ EFFECT_PLAYER_KEEP_RUNNING, {"Help My W Key Is Stuck", "player_keeprunning", true} },
+	{ EFFECT_VEH_WEAPONS, {"Vehicles Shoot Rockets (Left Click/RB)", "veh_weapons", true} },
+	{ EFFECT_MISC_AIRSTRIKE, {"Airstrike Inbound", "misc_airstrike", true} },
+	{ EFFECT_FAKE_PLAYER_SWAP, {"Player Swap", "player_playerswap", true, {}, true} },
+	{ EFFECT_PEDS_MINIONS, {"Minions", "peds_minions", true} },
+	{ EFFECT_PEDS_MERCENARIES, {"Mercenaries", "peds_mercenaries", true} },
+	{ EFFECT_LOOSE_TRIGGER, {"Loose Trigger", "peds_loosetrigger", true} },
+	{ EFFECT_MISC_FLAMETHROWER, {"Flamethrowers", "misc_flamethrower", true} },
+	{ EFFECT_MISC_DVDSCREENSAVER, {"DVD Screensaver", "misc_dvdscreensaver", true, {}, true} },
+	{ EFFECT_PLAYER_FAKEDEATH, {"Fake Death", "player_fakedeath"} },
+	{ EFFECT_GAMESPEED_SUPERHOT, {"Superhot", "time_superhot", true} },
+	{ EFFECT_PLAYER_KICKFLIP, {"Kickflip", "player_kickflip"} },
+	{ EFFECT_VEH_BEYBLADE, {"Beyblades", "vehs_beyblade", true, {}, true} },
+	{ EFFECT_KILLER_CLOWNS, {"Killer Clowns", "peds_killerclowns", true, {}, true} },
+	{ EFFECT_JAMES_BOND, {"Spawn Deadly Agent", "peds_jamesbond"} },
+	{ EFFECT_PLAYER_POOF, {"Deadly Aim", "player_poof", true} },
+	{ EFFECT_PLAYER_SIMEONSAYS, {"Simeon Says", "player_simeonsays", true, {}, true} },
+	{ EFFECT_VEH_LOCKCAMERA,  {"Lock Vehicle Camera", "veh_lockcamera", true} },
+	{ EFFECT_MISC_REPLACEVEHICLE, {"Replace Current Vehicle", "misc_replacevehicle"} },
+	{ EFFECT_PLAYER_TIRED,  {"I'm So Tired", "player_tired", true} },
+	{ EFFECT_MISC_SUPER_STUNT, {"Super Stunt", "misc_superstunt"} },
+	{ EFFECT_FLIP_CAMERA, {"Turn Turtle", "player_flip_camera", true} },
+	{ EFFECT_PLAYER_WALK_ON_WATER, {"Walk On Water", "player_walkonwater", true } },
+	{ EFFECT_RAPID_FIRE, {"Rapid Fire", "player_rapid_fire", true} },
+	{ EFFECT_PLAYER_ON_DEMAND_CARTOON, {"On-Demand TV", "player_on_demand_cartoon", true} },
+	{ EFFECT_PEDS_DRIVE_BACKWARDS, {"Peds Drive Backwards", "peds_drive_backwards", true} },
+	{ EFFECT_VEH_RANDTRAFFIC, {"Random Traffic", "veh_randtraffic", true, {}, true} },
+	{ EFFECT_MISC_RAMPJAM, {"Ramp Jam (Press Jump In Vehicle)", "misc_rampjam", true} },
+	{ EFFECT_MISC_VEHICLE_RAIN, {"Vehicle Rain", "misc_vehicle_rain", true, {}, true} },
+	{ EFFECT_MISC_CRASH, {"Fake Crash", "misc_fakecrash"} },
+	{ EFFECT_PLAYER_EASY_AIM, {"Bullet Time", "player_easy_aim", true, { EFFECT_GAMESPEED_X02, EFFECT_GAMESPEED_X05, EFFECT_GAMESPEED_SUPERHOT}} },
+	{ EFFECT_PLAYER_GRAVITY, {"Gravity Field", "player_gravity", true, {}, true} },
+	{ EFFECT_VEH_BOUNCY, {"Bouncy Vehicles", "veh_bouncy", true, {}, false} },
+	{ EFFECT_PEDS_STOP_AND_STARE, {"Stop and Stare", "peds_stop_stare"} },
+	{ EFFECT_PEDS_FLIP, {"Spinning Peds", "peds_flip", true, {}, true} },
+	{ EFFECT_PLAYER_PACIFIST, {"Pacifist", "player_pacifist", true, {}, false} },
+	{ EFFECT_VEH_TURN_RIGHT, {"Everyone Turn Right", "veh_turnright", true, {}, true} },
+	{ EFFECT_PEDS_BUSBOIS, {"Bus Bois", "peds_busbois"} },
```

Comment:
Lots of misc. whitespace. Was that intentional?

Reply:
Seems like Visual Studio messed something up.